src/Contracts/PhpSoap/Client.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace RicorocksDigitalAgency\Soap\Contracts\PhpSoap;
+
+use SoapHeader;
+
+interface Client
```

Comment:
Pretty sure it is possible to create a client using phpro/soap-client that implements this interface.